services/news/src/constants.ts

```diff
@@ -28,6 +28,7 @@ export const {
   GARDENING_HOST,
 
   // Ad-techs
+  DSP_HOST,
```

Comment:
Remove DSP_HOST. See detailed comment in EJS template.

Reply:
Done.